st2actions/st2actions/runners/python_action_wrapper.py

```diff
@@ -133,8 +133,12 @@ def run(self):
             action_output['status'] = action_status
             print_output = json.dumps(action_output)
         else:
-            raise InvalidStatusException('Status returned from the action must'
-                                         ' either be True or False.')
+            try:
```

Comment:
try / except is a bit redundant, you could just do `sys.exit` :)

It would also be a good idea to define exit code in some constants module. E.g. `PYTHON_ACTION_INVALID_STATUS_EXIT_CODE = 220` so it can be re-used across the code base.

Besides that, LGTM :+1: